### PR TITLE
explain what renaming a repository does

### PR DESCRIPTION
**Pull request by wshands**
Addresses dockstore/dockstore#4018

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -1,6 +1,20 @@
 FAQ
 ===
 
+What happens if I rename my GitHub repository?
+----------------------------------------------
+
+If you have registered a tool or workflow from GitHub in Dockstore the
```
Comment (coverbeck) [inline docs/faq.rst:7]: I suggest starting off this paragraph with with a version of of your second sentence -- it flows better from the title. Either that, or change "If" here to "When".

File: docs/faq.rst [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+link to the repository for your tool or workflow is displayed on the
+Info tab next to 'Source Code'. If you then `rename <https://docs.github.com/en/github/administering-a-repository/renaming-a-repository>`__
+the GitHub repository, the Source Code link will be incorrect, but
```
Comment (coverbeck) [inline docs/faq.rst:10]: I wouldn't say incorrect; I'd say it shows the old repository name.

File: docs/faq.rst [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+will resolve to the correct GitHub location when you click on it.
+
+Another side effect is that you will be able to register the workflow
```
Comment (coverbeck) [inline docs/faq.rst:13]: If you reuse the repo name, you're in bigger trouble. But GitHub itself councils against it.

**Review by coverbeck — APPROVED** [review]

File: docs/faq.rst (hunk 1 above)
Comment (wshands) [reply docs/faq.rst]: Changed If to When

File: docs/faq.rst (hunk 2 above)
Comment (wshands) [reply docs/faq.rst]: changed 'be incorrect' to 'display the original name'

File: docs/faq.rst (hunk 3 above)
Comment (denis-yuen) [reply docs/faq.rst:13]: Yeah, could be worth warning that you normally want to avoid this.

**Review by denis-yuen — APPROVED** [review]

**Review by Richard-Hansen — APPROVED** [review]

**Review by garyluu — APPROVED** [review]

File: docs/faq.rst (hunk 3 above)
Comment (wshands) [reply docs/faq.rst:13]: Added warning statement

**Pull request merged**